tests/test_dfm_metamod_model.py

```diff
@@ -0,0 +1,89 @@
+import textwrap
```

Comment:
why is this new? it was already existing right?

Reply:
maybe it was used in other parts of the program, but the imports have to be present in every file that uses them. textwrap is used in this file (line 42), so it has to be imported.